lua/lspconfig/server_configurations/ecsact.lua

```diff
@@ -0,0 +1,24 @@
+local util = require 'lspconfig.util'
+
+local bin_name = 'ecsact_lsp_server'
+local cmd = { bin_name, '--stdio' }
+
```

Comment:
```suggestion
if vim.fn.has 'win32' == 1 then
  cmd = { 'cmd.exe', '/C', bin_name, '--stdio' }
end

```

Reply:
what i mean is you should add windows support

Reply:
I understand. I was saying it wasn't required because it was not installed via gem or npm which is what the contributing doc said the reason it was required.

However I was mistaken in thinking that it didn't work with the `.appinstaller` version of the Ecsact LSP. I swear I tested it with and it didn't work, but it seemed to work now. ¯\\\_(ツ)_/¯

Sorry for the confusion!